app/controllers/application_controller.rb

```diff
@@ -13,7 +13,7 @@ class ApplicationController < ActionController::Base
   before_action :enforce_2fa
   before_action :block_write_request, if: :read_only_mode?
 
-  helper_method :top_level_post_types, :second_level_post_types
```

Comment:
You don't actually need this; helpers are available in views by default. You can remove this and just call `i18ns` directly in the views.

Reply:
removed, I see I can use `t` method too, so I changed `i18ns` to `t` .
Thanks